study-builder/studies/pancan/study-workflows.conf

```diff
@@ -40,31 +40,31 @@
           "type": "STUDY_REDIRECT",
           "studyGuid": "cmi-mpc",
           "redirectUrl": ${mpcRedirectUrl},
-          "expression": ${_pex.is_self}"&&"${_pex.has_prostate}"&&"${_pex.is_advanced_prostate},
+          "expression": ${_pex.is_self}"&&"${_pex.has_prostate}"&&"${_pex.is_advanced_prostate}"&&"${_pex.is_lang_en},
         },
         {
           "type": "STUDY_REDIRECT",
           "studyGuid": "ANGIO",
           "redirectUrl": ${angioRedirectUrl},
-          "expression": ${_pex.is_self}"&&"${_pex.has_angio},
+          "expression": ${_pex.is_self}"&&"${_pex.has_angio}"&&"${_pex.is_lang_en},
         },
```

Comment:
Can count on is_lang_en doing right thing? (I mean, do we have to worry that lang is not set at all, and then pex.is_lang_en returns "false" when, because en is the default language, it should return true?

Reply:
These users are temp users and while creating temp users, we fall back to english